File: math/primenumbers/485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let cnt=0
    let max=-Infinity
    for(let i=0;i<=nums.length-1;i++){
        if(nums[i]==0){
            cnt=0
        }
        else{
            cnt++
        }
        

        if(cnt>max){
            max=cnt
        }
    }

    return max
};

File: math/primenumbers/485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxcnt=Integer.MIN_VALUE;
        int cnt=0;

        for(int i=0;i<nums.length;i++){
            if(nums[i]==1){
                cnt++;
            }
            else{
                cnt=0;
            }

            maxcnt=Math.max(maxcnt,cnt);
        }

        return maxcnt;
        
    }
}